public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -478,21 +498,63 @@ export function AnomalyResults(props: AnomalyResultsProps) {
                         <EuiButton
                           color={isDetectorFailed ? 'danger' : 'warning'}
                           onClick={props.onStartDetector}
-                          iconType={'play'}
                           style={{ marginLeft: '8px' }}
                         >
                           Restart detector
                         </EuiButton>
                       ) : null}
                     </EuiCallOut>
                   ) : null}
+                  {detector ? (
+                    <EuiFlexGroup
+                      justifyContent="spaceBetween"
+                      style={{ marginBottom: '18px' }}
+                    >
+                      <EuiFlexItem grow={false}>
+                        <EuiTitle size="m">
+                          {
+                            <h1>
+                              {'Real-time results'}{' '}
+                              {getDetectorStateDetails(
+                                detector,
+                                isHCDetector,
+                                false,
+                                false
+                              )}
+                            </h1>
+                          }
+                        </EuiTitle>
+                      </EuiFlexItem>
+
+                      <EuiFlexItem grow={false}>
+                        <EuiButton
+                          data-test-subj="stopDetectorButton"
+                          onClick={
+                            detector?.enabled
+                              ? props.onStopDetector
+                              : props.onStartDetector
+                          }
+                          disabled={
+                            !detector?.featureAttributes ||
+                            detector?.featureAttributes.length === 0
```

Comment:
It's a good point - from UX side I know it is better to allow buttons to be clicked and see the following errors, rather than disabling buttons and having customers guess why something is disabled.

In fact, it is probably better to not disable this button at all, regardless of features created or not. Let me confirm with UX.

What do you think?

Reply:
Also, this is the current error when trying to start a detector where all features are disabled: 
![Screen Shot 2021-07-19 at 3 08 49 PM](https://user-images.githubusercontent.com/62119629/126234098-1d2b999b-5fb3-4a32-8244-ff4ca720583a.png)

Reply:
I've confirmed with UX about removing this disabled functionality at all, and to always leave in enabled

Reply:
so we will not allow cx to disable features any more?

Reply:
No, they can still disable features. This is only changing the start/stop button on the results page. Now it is never disabled, so users can always try to start/stop, but may get errors like the example above (no enabled features, etc.)

Reply:
got it.

Reply:
got it.